lib/memory/index.js

```diff
@@ -173,11 +173,22 @@ module.exports = function MemoryStore(data_set) {
   )
 
 
+  const del = R.curry((bucket, id) =>
```

Comment:
Change all of these `del` functions of the Kuss interfaces to `deleteById` to be more explicit